corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -188,14 +188,23 @@
 <div class="row">
     <div class="col-sm-12">
         <div class="btn-toolbar">
-            <a class="btn btn-primary"
+            <a class="btn btn-info"
+               {% if not has_location_types or not can_edit_any_location %}
```

Comment:
I'm not familiar with the location permissions, but is it possible for someone to be able to view, but not edit locations?